wal_test.go

```diff
@@ -965,3 +965,31 @@ func TestClose(t *testing.T) {
 	err = w.SetUint64([]byte("foo"), 1)
 	require.ErrorIs(t, err, ErrClosed)
 }
+
+func TestGetSealedSegments(t *testing.T) {
+	tsOpts := []testStorageOpt{
+		segFull(),
+		segFull(),
+		segFull(),
+		segFull(),
+		segTail(10),
+	}
+
+	store := makeRaftLogs(411, 5)
+	ts, w, err := testOpenWAL(t, tsOpts, nil, false)
+	require.NoError(t, err)
+
+	err = w.StoreLogs(store)
+	require.NoError(t, err)
+
+	t.Log(ts.debugDump())
+
+	sealed, err := w.GetSealedLogFiles(200)
+	if err != nil {
+		t.Fatalf("failed to get sealed logs: %v", err)
+	}
+
+	if len(sealed) != 3 {
```

Comment:
Why 3 rather than 4?  I gather it has to do with the fromIndex=200, but if you could add a hint as a comment it would be nice.

Reply:
Yeah, exactly, fromIndex points to a baseIndex of a segment which is included in the result, and the sum is only three segments. The last one is the tail which is not sealed yet.

Reply:
I will add a comment!